src/main/resources/jupyter/init-actions.sh

```diff
@@ -14,6 +14,30 @@ function update_apt_get() {
   return 1
 }
 
+function run_docker_compose() {
+  for ((i = 0; i < 5; i++)); do
+    docker-compose -f /etc/cluster-docker-compose.yaml config
```

Comment:
qq, if `config` throws error, retry wouldn't make successful, right?

Reply:
That's a good point -- shouldn't need to retry `config`